### PR TITLE
Small code cleanup

### PR DESCRIPTION
**Pull request by fneum**
- remove unused imports
- remove unused variables
- improve type checks
- remove reimports
- misc code style issues

### CODE REVIEW

**Comment by fneum** [comment]
Where are the `from __future__ import ...` still necessary and where not?

**Comment by FabianHofmann** [comment]
The `__future__` imports are all referring to compatibility between python 2 and python 3. As far as I see there remain, the following future imports:

* `absolute_import` 
* `print` 
* `division`

All of those are in the [MandatoryRelease](https://docs.python.org/3/library/__future__.html) of python 3, meaning they are not needed for version > 3.

**Comment by FabianHofmann** [comment]
I'll clean them out and push it here

**Comment by FabianHofmann** [comment]
Also not needed anymore:

* `from six import string_types` &rarr; use standard `str` instead
* `from six.moves import range` &rarr; use standard `range`
* `from six.moves import map` &rarr; use standard `map`

Often unnecessary 
* `from six import iteritems` and `iterkeys` &rarr; use the .items() function for dicts and series

But perhaps we keep this for another PR, since it requires modifying the code a bit

File: test/test_pf_against_pandapower.py [hunk 1]
```diff
@@ -17,66 +12,89 @@
...
+    )
+    np.testing.assert_array_almost_equal(
+        n.transformers_t.p1.loc["now"], net.res_trafo.p_lv_mw
+    )
+    np.testing.assert_array_almost_equal(
+        n.transformers_t.q0.loc["now"], net.res_trafo.q_hv_mvar
+    )
+    np.testing.assert_array_almost_equal(
+        n.transformers_t.q1.loc["now"], net.res_trafo.q_lv_mvar
+    )
```
Comment (FabianHofmann) [inline test/test_pf_against_pandapower.py]: I guess this is only line breaks or?

Comment (fneum) [reply test/test_pf_against_pandapower.py]: Yes, I have mapplied black coding style to this script as I was unhappy with the formatting of lines 41ff.

**Comment by fneum** [comment]
Thanks for going through the examples! It's a bit odd that you need the `cufflinks` although it is never really called anywhere.

**Comment by FabianHofmann** [comment]
yes, it's because some interactive are called via `pd.DataFrame.iplot()`. This does not come with the standard pandas package, cufflinks has to be imported to enable it

**Comment by fneum** [comment]
Will merge anyway since failing PR CI is only due to Mac hickups.

**Pull request merged**